base-server/index.js

```diff
@@ -54,11 +55,16 @@ export class LoguxNotFoundError extends Error {
 
 function normalizeTypeCallbacks(name, callbacks) {
   if (callbacks && callbacks.accessAndProcess) {
-    callbacks.access = (...args) => {
+    callbacks.access = (ctx, action, meta) => {
```

Comment:
We also can do
```suggestion
    callbacks.access = (ctx, ...args) => {
```

saves a little space